components/SplitsDisplay.vue

```diff
@@ -76,14 +80,14 @@ import {
   selectTime,
   SplitFile,
   splitFileIsModified
-}                              from '~/util/splits';
-import {Vue, Component, Watch} from 'nuxt-property-decorator';
-import {stringTimeToSeconds}   from '~/util/durations';
-import {xmlParser}             from '~/util/xml';
-import VueSlider               from 'vue-slider-component';
-import {whithLoadAsync}        from '~/util/loading';
-import {asArray}               from '~/util/util';
-import store                   from '~/util/store';
+}                                    from '~/util/splits';
+import {Vue, Component, Watch, Prop} from 'nuxt-property-decorator';
```

Comment:
As Sonar suggested, this `Prop` import isn't useful, you can remove it (and rollback the whole whitespace changes on all the imports)